settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'dal'
+include 'cadc-dali', 'cadc-dali-pg', 'cadc-datalink-server', 'cadc-sia', 'cadc-soda-server'
```

Comment:
What is this settings.gradle for?

Reply:
This is used at the top level by Gradle.  If, while in the root of the project, I run `gradle -i clean build test`, it will iterate over the `include` projects and run those gradle tasks on them.  It good for making sure that I didn't break anything else in the repository when I make a change.  It may simplify the `.travis.yml` file, too.